npm run build fails on windows

Are you experiencing an issue with...

- [x] My own instance

## :beetle: **Description**

`npm run build` fails on windows:
```console
λ npm run build

> shields.io@0.0.0 prebuild G:\shields
> run-s --silent depcheck


> shields.io@0.0.0 build G:\shields
> run-s defs features && gatsby build


> shields.io@0.0.0 defs G:\shields
> node scripts/export-service-definitions-cli.js > service-definitions.yml


> shields.io@0.0.0 features G:\shields
> node scripts/export-supported-features-cli.js > supported-features.json

success open and validate gatsby-configs — 0.022 s
success load plugins — 0.449 s
success onPreInit — 0.388 s
success delete html and css files from previous builds — 0.038 s
success initialize cache — 0.018 s
success copy gatsby files — 1.107 s
success onPreBootstrap — 0.059 s
success source and transform nodes — 0.082 s
success building schema — 0.647 s
success createPages — 0.118 s
error [G:/shields/frontend/pages/endpoint.spec.js] The page component must export a React component for it to be valid
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! shields.io@0.0.0 build: `run-s defs features && gatsby build`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the shields.io@0.0.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\[USERNAME]\AppData\Roaming\npm-cache\_logs\2019-03-17T20_45_37_978Z-debug.log
```

Seems to be related to #3208

--- a/package.json
+++ b/package.json
@@ -161,7 +161,7 @@
     "fs-readfile-promise": "^3.0.1",
     "gatsby": "2.1.19",
     "gatsby-plugin-catch-links": "^2.0.13",
-    "gatsby-plugin-page-creator": "^2.0.10",
+    "gatsby-plugin-page-creator": "^2.0.12",
     "gatsby-plugin-react-helmet": "^3.0.10",
     "gatsby-plugin-remove-trailing-slashes": "^2.0.10",
     "gatsby-plugin-styled-components": "^3.0.7",

